ddraw/DebugOverlay.cpp

```diff
@@ -98,7 +99,7 @@ void DebugOverlay::BeginScene()
 	ImGui::NewFrame();
 }
 
-void DebugOverlay::EndScene()
+void DebugOverlay::EndScene(const RenderData &RenderData)
```

Comment:
It looks like `RenderData` is never used in the DebugOverlay.  So there is no need to pass it.

Reply:
Let's keep this out until we are actually using the data in the overlay.